### ISSUE TITLE
Increase prometheus readiness failureThreashold via helm values

### ISSUE THREAD

**Issue by wobkenh** [rancher/charts#1635]
Hey there,

we are deploying the rancher-monitoring chart in several of our rke clusters.
In one of our clusters, prometheus refuses to start within the failureThreshold of 120 seconds because it wants to restore its database state from the postgres WAL, which takes longer than the specified 120 seconds. 

We could not find a way to increase the failureThreshold via helm chart. We also wondered where the rancher-prometheus-monitoring statefulset that includes said readinessProbe is generated from because we can see it in our clusters, but if we `helm template` the rancher-monitoring chart, we cant find the corresponding yaml in the output.

Long story short - is there a way to increase the failureThreshold of the prometheus statefulset (see below) using the values of the rancher-monitoring helm chart?

We are using chart version 16.6.0 with mirrored-prometheus-prometheus:v2.27.1.

```
readinessProbe:
  failureThreshold: 120
  httpGet:
    path: /-/ready
    port: nginx-http
    scheme: HTTP
  periodSeconds: 5
  successThreshold: 1
  timeoutSeconds: 3
```

**Comment by 8enmann** [comment]
bump

**Comment by aiyengar2** [comment]
@wobkenh sorry for the late response! This does not seem configurable today since it's an upstream issue; it's hard-coded in the Prometheus Operator: https://github.com/prometheus-operator/prometheus-operator/blob/bb7ec39d215062b6fbb2398547ed8c9b74d93830/pkg/alertmanager/statefulset.go#L322

Try filing an issue upstream to allow this to be configured in the Prometheus CR spec and we can try to absorb the changes!
